### DISCUSSION TITLE
What is FFMPEG P-ID?

### DISCUSSION THREAD

**Discussion by dz4news** [blakeblackshear/frigate#7658] [General]
I am having an issue with one camera "backyard" and it seems to giving errors that are network related. Today I noticed the FFMPEG PID value is much higher than the other cameras. Is this a concern? I cant find much info about P-ID and what it indicates if anything.
<img width="1153" alt="Screenshot 2023-09-02 at 1 27 31 PM" src="https://github.com/blakeblackshear/frigate/assets/57462035/e1194f0a-ab2c-406b-88e7-a15ff6ed85b6">

**Comment by NickM-27** [comment]
PID is simply the ID of the process, as frigate is open longer and a stream fails, the ID that is given to that process. It is not an issue except a symptom of the process being restarted many times
